universal-application-tool-0.0.1/conf/evolutions/default/35.sql

```diff
@@ -0,0 +1,7 @@
+# --- Store optional user_file_name with file names
+
+# --- !Ups
+alter table files add column user_file_name varchar;
```

Comment:
does this not need some optional option

Reply:
it's the other way around. everything is optional/nullable unless we add a "not null" constraint to make it required.